### ISSUE TITLE
ObjectDisposedException when exiting Player

### ISSUE THREAD

**Issue by smoogipoo** [ppy/osu#6362]
```
System.ObjectDisposedException: Can not start disposed tracks.
Object name: 'osu.Framework.Audio.Track.TrackBass'.
   at osu.Framework.Audio.Track.Track.Start() in C:\projects\osu-framework-a4n7e\osu.Framework\Audio\Track\Track.cs:line 105
   at osu.Framework.Audio.Track.TrackBass.Start() in C:\projects\osu-framework-a4n7e\osu.Framework\Audio\Track\TrackBass.cs:line 228
   at osu.Framework.Timing.DecoupleableInterpolatingFramedClock.Start() in C:\projects\osu-framework-a4n7e\osu.Framework\Timing\DecoupleableInterpolatingFramedClock.cs:line 129
   at osu.Framework.Timing.DecoupleableInterpolatingFramedClock.ProcessFrame() in C:\projects\osu-framework-a4n7e\osu.Framework\Timing\DecoupleableInterpolatingFramedClock.cs:line 99
   at osu.Framework.Timing.FramedClock.ProcessFrame() in C:\projects\osu-framework-a4n7e\osu.Framework\Timing\FramedClock.cs:line 73
   at osu.Framework.Timing.FramedClock.ProcessFrame() in C:\projects\osu-framework-a4n7e\osu.Framework\Timing\FramedClock.cs:line 73
   at osu.Game.Screens.Play.GameplayClockContainer.Update() in /Users/smoogipoo/Repos/osu/osu.Game/Screens/Play/GameplayClockContainer.cs:line 165
   at osu.Framework.Graphics.Drawable.UpdateSubTree() in C:\projects\osu-framework-a4n7e\osu.Framework\Graphics\Drawable.cs:line 413
   at osu.Framework.Graphics.Containers.CompositeDrawable.UpdateSubTree() in C:\projects\osu-framework-a4n7e\osu.Framework\Graphics\Containers\CompositeDrawable.cs:line 802
   at osu.Framework.Graphics.Containers.CompositeDrawable.UpdateSubTree() in C:\projects\osu-framework-a4n7e\osu.Framework\Graphics\Containers\CompositeDrawable.cs:line 815
   at osu.Framework.Graphics.Containers.CompositeDrawable.UpdateSubTree() in C:\projects\osu-framework-a4n7e\osu.Framework\Graphics\Containers\CompositeDrawable.cs:line 815
```

**Comment by smoogipoo** [comment]
May be fixed by #6344

**Comment by peppy** [comment]
Haven't noticed this happen in recent times, so I think it's been alleviated since.
